libfwupdplugin/fu-kernel.c

```diff
@@ -466,3 +466,113 @@ fu_kernel_get_cmdline(GError **error)
 	return NULL;
 #endif
 }
+
+gboolean
+fu_kernel_check_cmdline_mutable(GError **error)
+{
+	const gchar *config_files[] = {"/boot/grub2/grub.cfg", "/etc/grub2.cfg", NULL};
```

Comment:
use the new path detecting for `/boot` and also use the sysfconf dir detection for `/etc`

Reply:
Fixed.